serverlessworkflow/antora.yml

```
@@ -43,16 +45,20 @@ asciidoc:
     operator_openshift_catalog: sonataflow-operator
     operator_k8s_keyword: sonataflow
     operator_k8s_subscription: my-sonataflow-operator
-    kogito_devservices_imagename: quay.io/kiegroup/kogito-data-index-ephemeral-nightly
-    sonataflow_devmode_imagename: quay.io/kiegroup/kogito-swf-devmode-nightly
-    sonataflow_builder_imagename: quay.io/kiegroup/kogito-swf-builder-nightly
+    container_image_registry_base: docker.io
+    sonataflow_operator_imagename: docker.io/apache/incubator-kie-sonataflow-operator
+    sonataflow_devmode_imagename: docker.io/apache/incubator-kie-sonataflow-devmode
+    sonataflow_builder_imagename: docker.io/apache/incubator-kie-sonataflow-builder
+    sonataflow_builder_configmap_name: sonataflow-operator-builder-config
+    sonataflow_dataindex_postgresql_imagename: docker.io/apache/incubator-kie-kogito-data-index-postgresql
+    sonataflow_dataindex_ephemeral_imagename:  docker.io/apache/incubator-kie-kogito-data-index-ephemeral
     sonataflow_devmode_devui_url: /q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/
     serverless_logic_web_tools_name: Serverless Logic Web Tools
     serverless_workflow_vscode_extension_name: KIE Serverless Workflow Editor
     kie_kogito_examples_repo_name: incubator-kie-kogito-examples
 
     # Jobs service image and links
-    jobs_service_image_allinone_url: https://hub.docker.com/r/apache/incubator-kie-kogito-jobs-service-allinone
+    jobs_service_image_allinone_url: https://hub.docker.com/r/apache/incubator-sonataflow-data-index-postgresql
```

```suggestion
    jobs_service_image_allinone_url: https://hub.docker.com/r/apache/incubator-sonataflow-data-index-postgresql
```
This image can be removed from docs. We will deprecate/delete it. cc @wmedvede

---

But most, incubator-sonataflow-data-index-postgresql image does not exist.

---

Yea, will fix thanks

---

This is public GH space @gmunozfe